game/scenes/MainMenu.gd

```diff
@@ -5,6 +5,9 @@ extends Control
 @export_file("*.tscn") var credits_scene: String
 @export_file("*.tscn") var help_scene: String
 
+func _ready() -> void:
+	BackgroundMusic.play_menu_music()
```

Comment:
if the level scene does a stop before playing it's music I assume we should do the same here, in case the user lost and returned to menu, we should stop the player before playing menu music.

Alternatively, both "play" methods could stop previously played tracks before playing the new ones.


another option could be to call "stop" before switching to main menu on the level scene itself

Reply:
This stop is because the music stops when Mecha-Peron appears in the screen for the intro part. Then it resumes when the console is dismissed